lib/i18n/hygiene/task.rb

```diff
@@ -0,0 +1,58 @@
+require 'rake'
+require 'rake/tasklib'
+require 'i18n/hygiene/config'
+require 'i18n/hygiene/reporter'
+require 'i18n/hygiene/checks/missing_interpolation_variable_check'
+
+module I18n
+  module Hygiene
+    class Task < ::Rake::TaskLib
```

Comment:
`cane` and `rubocop` call their version of this class `RakeTask`. I like they way that signals what the class is for, what do you think?